src/util/CMakeLists.txt

```diff
@@ -3,7 +3,7 @@ unset (util_SOURCES)
 file (GLOB_RECURSE util_SOURCES *.cpp)
 
 add_library (util STATIC ${util_SOURCES})
-target_link_libraries(util PUBLIC std::filesystem cxx17)
+target_link_libraries(util PUBLIC cxx17)
```

Comment:
I would leave this here, since we need it there anyways when switching completely to std::filesystem.